### ISSUE TITLE
Implementing feature for pdf to docx conversion

### ISSUE THREAD

**Issue by mishraaditya595** [mishraaditya595/ScanIt#15]
Most online pdf to doc convertors use OCR to do the job. OCR has already been implemented in the app. A feature to convert pdf files into docx can be done.

**Comment by tharunc** [comment]
Hello there,
    I would like to add work on this issue. Kindly assign me this issue as a part of DWOC.

**Comment by kunal299** [comment]
@mishraaditya595 I would also like to work on this issue. Kindly assign me this issue when the GSSoC starts.

**Comment by mishraaditya595** [comment]
I have not received any notice from the organisers to start assigning issues. I will do it as soon as I get confirmation from their side. Glad to see your interest.

**Comment by kunal299** [comment]
@mishraaditya595 sure

**Comment by subhangi2731** [comment]
can you please assign this to me?

**Comment by Shaswat-2203** [comment]
yeah , mentors told us to go to projects we like and ask you all to assign in particular issue

**Comment by sudhansukinng** [comment]
@mishraaditya595 sir, Please assign this issue to me. I would love to work on this issue.

**Comment by kunal299** [comment]
@mishraaditya595 I am working on this issue.

**Comment by mishraaditya595** [comment]
Hey @kunal299, what is the status of your work? It has been 6 days. If you are facing some issues, let me know.

**Comment by kunal299** [comment]
@mishraaditya595 I was just researching about ocr and learning how to implement it. As I am new to android dev, could you just break down the tasks for this issue so that i would complete it faster.

**Comment by mishraaditya595** [comment]
Ok! That's cool. I will try to simplify it. In the meantime, does anyone else also want to work on it? @sudhansukinng @Shaswat-2203 @subhangi2731

**Comment by Shaswat-2203** [comment]
Yeah sure I want to work in it @mishraaditya595
